work/flow.rst

```diff
@@ -478,6 +478,11 @@ Putting a ticket in a **Reviewed** state gives the developer the go-ahead to mer
 If it has not been done already, the developer should rebase the ticket branch against the latest master.
 If a rebase was required, a final check with Jenkins should be done.
 
+**GitHub pull request pages offer a 'big green button' for merging a branch to master**.
+We encourage you to use this button when GitHub says "This branch has no conflicts with the base branch", "All checks have passed", and at least one of the checks has "Required" next to it, which should be the case for almost all repos that are being changed.
+For normally-configured repos, using the button will also delete the ticket branch after the merge.
+Do *not* select a different merge strategy from the pulldown next to the button; these should be disabled anyway.
```

Comment:
May want to say "and do not hit the Update my Branch" button since this will merge master to your branch and will not rebase it.